index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = algoliasearch;
+
+function algoliasearch(applicationID, apiKey, opts) {
+  var AlgoliaSearch = require('./src/algoliasearch');
+
+  return new AlgoliaSearch(applicationID, apiKey, opts);
```

Comment:
You mean https://github.com/algolia/algoliasearch-client-js/blob/new-usage-40/src/algoliasearch.js#L28-L43?